lib/parse-args.js

```diff
@@ -191,18 +189,13 @@ function buildYargs (withCommands = false) {
 function hideInstrumenterArgs (yargv) {
   let argv = process.argv.slice(1)
   argv = argv.slice(argv.indexOf(yargv._[0]))
-  if (argv[0][0] === '-') {
-    argv.unshift(process.execPath)
-  }
   return argv
 }
 
 function hideInstrumenteeArgs () {
   let argv = process.argv.slice(2)
   const yargv = parser(argv)
 
-  if (!yargv._.length) return argv
```

Comment:
`yargv._` is this a call to underscore?  Is there another Test Case I need to add for this condition?